security/policykit/Portfile

```diff
@@ -38,7 +35,7 @@ depends_build-append \
                     port:perl5.28
 
 depends_lib         path:lib/pkgconfig/glib-2.0.pc:glib2 \
-                    port:mozjs60
+                    port:mozjs78
```

Comment:
This suggestion is outside your responsibilities, but it would be nice to keep around the old policykit (which relies on Rust-free `mozjs60`) for 10.4/10.5/10.6 users. We do something similar for `librsvg`.